addons/im_livechat/static/src/core/web/livechat_channel_info_list.js

```diff
@@ -55,6 +71,13 @@ export class LivechatChannelInfoList extends Component {
         });
     }
 
+    onClickEditTags(ev) {
+        this.tagEditPopover.open(this.tagsContainer.el, {
+            thread: this.props.thread,
+            selectedTags: this.props.thread.livechat_conversation_tag_ids,
```

Comment:
Or I guess it's more this part that's not reactive, the selectedTags should be updated when livechat_conversation_tag_ids are changing